earth_enterprise/docs/landing_page/manual/answer/4556704.html

```diff
@@ -4,7 +4,7 @@
 <!-- 4556704.html -->
 <div class="container">
   <div class="sidebar1">
-<p class="sidebar-toc"><a href="../answer/7159245.html">GEE 5.1.3 Beta</a></p>
```

Comment:
Any changes to the documentation here will also need to be made on the gh-pages branch.